src/components/organisms/proposal-summary.tsx

```diff
@@ -89,3 +101,82 @@ export const ProposalSummary = ({
     </Card>
   )
 }
+
+// TODO: https://github.com/regen-network/regen-js/issues/71
+function renderMessage(msg: any, groupPolicyAddress: string) {
```

Comment:
This also gives us a lint warning but we should have a specific type that works correctly and would be temporary.

Reply:
Same thing. feel free to disable. Alternatively, if these are the same type you can define that somewhere else and use it, like 
```tsx
// eslint-disable-next-line @typescript-eslint/no-explicit-any
type MsgAnyType = any
```
so you only need to disable the linter once.

Either works